### PR TITLE
Dtspb 2054 dispense notice

### PR DESCRIPTION
**Pull request by DelythJustice**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/service/tasklist/TaskStateRenderer.java [hunk 1]
```diff
@@ -159,28 +163,31 @@ private String renderSendDocsDetails(TaskState sendDocsState, String caseId, Cas
...
                 DetailsComponentRenderer.renderByReplace(SEND_DOCS_DETAILS_TITLE,
                         SendDocumentsDetailsHtmlTemplate.DOC_DETAILS.replaceFirst("<refNum/>", caseId)
-                .replaceFirst(ORIGINAL_WILL, keyValues.getOrDefault("originalWill", ""))
-                .replaceFirst(IHT_TEXT, keyValues.getOrDefault("ihtText", ""))
-                .replaceFirst(IHT_FORM, keyValues.getOrDefault("ihtForm", ""))
-                .replaceFirst(RENOUNCING_EXECUTORS, keyValues.getOrDefault("renouncingExecutors", ""))
-                .replaceFirst(PA16_FORM, keyValues.getOrDefault("pa16Form", ""))
-                .replaceFirst(PA17_FORM, keyValues.getOrDefault("pa17Form", ""))
-                .replaceFirst(IHT_ESTATE_207, keyValues.getOrDefault("ihtEstate207", ""))
+                            .replaceFirst(ORIGINAL_WILL, keyValues.getOrDefault("originalWill", ""))
```
Comment (sanjayparekh-hmcts) [inline src/main/java/uk/gov/hmcts/probate/service/tasklist/TaskStateRenderer.java]: can you adjust your formating to be the same as current please, so we dont have these big changes. And then apply to the whole of your changeset

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]
Rest of it looks good to me

File: README.md [hunk 2]
```diff
@@ -22,8 +22,8 @@ The template is a working application with a minimal setup. It contains:
  * Hystrix dashboard
  * MIT license and contribution information
 
-The application exposes health endpoint (http://localhost:4550/health) and metrics endpoint
-(http://localhost:4550/metrics).
+The application exposes health endpoint (http://localhost:3455/health) and metrics endpoint
+(http://localhost:3455/metrics).
```
Comment (nigeldunne) [inline README.md]: isn't BO running on 4104?

**Review by nigeldunne — COMMENTED** [review]
Do we need any fuinctional tests?

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Pull request merged**